frontend/src/assets/main.scss

```diff
@@ -16,6 +16,14 @@
   }
 }
 
+.content-center {
+  text-align: center !important;
+}
+
 .content-right {
   text-align: right !important;
 }
+
+.permissions-modal {
+  width: 800px;
```

Comment:
this seems risky. might be better to do something like `max-width: 800px; width: 80%`